gapis/api/vulkan/api/enums.api

```diff
@@ -283,6 +283,15 @@ enum VkStructureType: u32 {
   //@extension("VK_EXT_hdr_metadata")
   VK_STRUCTURE_TYPE_HDR_METADATA_EXT = 1000105000,
 
+  // extension("VK_KHR_create_renderpass2")
+  VK_STRUCTURE_TYPE_ATTACHMENT_DESCRIPTION_2 = 1000109000,
+  VK_STRUCTURE_TYPE_ATTACHMENT_REFERENCE_2 = 1000109001,
+  VK_STRUCTURE_TYPE_SUBPASS_DESCRIPTION_2 = 1000109002,
+  VK_STRUCTURE_TYPE_SUBPASS_DEPENDENCY_2 = 1000109003,
+  VK_STRUCTURE_TYPE_RENDER_PASS_CREATE_INFO_2 = 1000109004,
```

Comment:
Why is this in a different order than the spec? (Also alphabetically)

Reply:
The order here does not matter. The important bit is the numbers. Lots of enums we have are in different order than the spec. If it's an issue(I don't think it is) we can have a one-off PR to clean them up. The general problem is that, we always add it to last when we add a new extension but that does not necessarily reflect the spec. I have tried to put it to somewhere that more or less where it should be.